src/fsevent.rs

```diff
@@ -531,14 +534,14 @@ unsafe fn callback_impl(
         for ev in translate_flags(flag, true).into_iter() {
             // TODO: precise
             let ev = ev.add_path(path.clone());
-            (*event_fn)(Ok(ev));
+            (event_fn.lock().unwrap())(Ok(ev));
```

Comment:
I'd use an expect with a declaration what failed.

Reply:
Switched to `expect()`

Reply:
thanks, I meant more something about event_fn having this poisoning, but maybe that's not good